src/reducers/entities/__snapshots__/apps.test.ts.snap

```diff
@@ -0,0 +1,324 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`bindings Invalid channel header get filtered 1`] = `
```

Comment:
Using a snapshot for something that isn't React looks a little weird to me, but I don't see any major reason not to do it at the moment. It might be better for the tests to check specifically for what they want changed, but I'm 0/5 on that.

Reply:
My reasoning to use snapshots is that it was simpler and clearer to see what elements are filtered and what elements are not, and how the information is moved around. The alternative was to use something like "equals", and define both the input and the output in the body of the text, which would pollute heavily the code.

Nevertheless, I am more than happy to hear about other alternatives :smile: